models/migrations/migrations.go

```diff
@@ -342,6 +342,8 @@ var migrations = []Migration{
 	NewMigration("Add repo id column for attachment table", addRepoIDForAttachment),
 	// v194 -> v195
 	NewMigration("Add Branch Protection Unprotected Files Column", addBranchProtectionUnprotectedFilesColumn),
+	// v196 -> v197
```

Comment:
Shouldn't that be
```suggestion
	// v195 -> v196
```
?

Reply:
Ah, yes, typo

Reply:
see #17130.